cfy_manager/components/manager/manager.py

```diff
@@ -20,16 +20,22 @@
 
 from ..base_component import BaseComponent
 from ..service_names import MANAGER
-from ..components_constants import CONFIG
+from ..components_constants import CONFIG, SERVICES_TO_INSTALL
+from ..service_components import QUEUE_SERVICE
+from ..service_names import (
+    RABBITMQ,
+)
+from ...config import config
```

Comment:
This should be after the `from ... import constants`

Reply:
Updated